web/modules/custom/dpl_example_content/content/media/618e176a-a45a-4c36-a197-664230aa0a34.yml

```diff
@@ -33,6 +33,10 @@ default:
       attributes:
         rel: canonical
         href: '/user/login'
+  path:
+    -
+      alias: ''
+      langcode: da
```

Comment:
It seems this config is added to most example contents we have as part of this commit. 
Just highlighting since i wasn't sure if this was intentional. I guess its' just something drupal picked up on as part of updating the content?

Reply:
It's coming from the `dce drush default-content:export-module dpl_example_content` and we don't want to cherry-pick what it does